plugins/installer/override/override.php

```diff
@@ -337,19 +336,26 @@ private function saveOverrides($pks)
 		{
 			$insertQuery->clear('values');
 
-			$createdDate = $today->format('y-m-d h:i:s');
+			$tz = new \DateTimeZone(Factory::getApplication()->get('offset'));
+			$date = new Date('now');
+			$date->setTimezone($tz);
+			$createdDate = $date->toSql(true);
 
-			if (empty($pk->modifiedDate))
+			if (empty($pk->coreFile))
 			{
-				$pk->modifiedDate = '0000-00-00 00:00:00';
+				$modifiedDate = '0000-00-00 00:00:00';
```

Comment:
Here we can take the null date from the database driver to stay on the safe side.

Reply:
@astridx Can you please take a look?

Reply:
@laoneo Do you mean this: https://github.com/joomla-projects/gsoc18_override_management/pull/52/commits/2698f349ef659f70159b77713bb366c62e221467